cypress/integration/twilio-video.spec.js

```diff
@@ -82,6 +82,38 @@ context('A video app user', () => {
       cy.getParticipant('test1').should('not.exist');
       cy.get('[data-cy-main-participant]').should('contain', 'testuser');
     });
+
+    describe('the recording start/stop feature', () => {
+      before(() => {
+        cy.get('[data-cy-more-button]')
+          .last()
+          .click();
+        cy.get('[data-cy-recording-button]').click();
+      });
+
+      after(() => {
+        cy.wait(3000);
+      });
+
+      it('should see the recording indicator and notification after clicking "Start Recording"', () => {
+        cy.get('[data-cy-recording-indicator]').should('be.visible');
+        cy.contains('Recording has started').should('be.visible');
+        cy.wait(2000);
```

Comment:
I think this wait statement should go at the end of the `before()` block. Looks like there isn't a delay between recording start (in the before block) and recording stop (in the "it should see 'Recording Complete' notification" test).